ai/core/tests/core/databricks/test_databricks_integration_tests.py

```diff
@@ -381,7 +381,7 @@ def simple_func(x: int) -> str:
         # Attempt to create the same function again without replace
         with pytest.raises(
             Exception,
-            match=f"Cannot create the function `{CATALOG}`.`{SCHEMA}`.`simple_func` because it already exists",
+            match=f"`{CATALOG}`.`{SCHEMA}`.`simple_func` because it already exists",
```

Comment:
Change this because the error message changed:
```
[ROUTINE_ALREADY_EXISTS] Cannot create the routine ...
```
This is spark error so we have no control, drop matching them